src/System.Diagnostics.Process/tests/ProcessStandardConsoleTests.cs

```diff
@@ -43,22 +43,12 @@ public void TestChangesInConsoleEncoding()
 
             try
```

Comment:
ok, will try to add cases for non-ASCII considering setting the codepage on the console.